### ISSUE TITLE
Unexpected result when copy/pasting file with very large number

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#50823]
Saw while testing #47014

- Have a file with a large number in the name, like `file-123123123123123123213123123123.txt`
- Copy/paste it in the explorer
- Get a file name formatted as a large JS number, like `file-000000001.2312312312312312e+29.txt` and if you copy/paste that, the inner number increments.

Sort of related to https://github.com/Microsoft/vscode/issues/50822 but this is not a regression and not a big problem.

**Comment by isidorn** [comment]
fyi @Krzysztof-Cieslak

**Comment by Krzysztof-Cieslak** [comment]
Yeah, this is definitely not regression- it probably behaved exactly the same with old implementation. I’ll take a look.

**Comment by isidorn** [comment]
@Krzysztof-Cieslak great, no rush. https://github.com/Microsoft/vscode/issues/50822 is the only one I plan to fix today, or if you want you can also fix it

**Comment by Krzysztof-Cieslak** [comment]
I think this actually got also fixed by https://github.com/Microsoft/vscode/pull/50865 - since we check for max safe integer, there is no chance to use number big enough to cause wrong number formatting.
